### PR TITLE
fix jshint indentified issue for all studio and edx_sga file

### PR DESCRIPTION
**Pull request by amir-qayyum-arbisoft**
In this PR i fixed jshint identified issues. list of issues are attached in image.

#84 

jshint_error_on_edxsga
jshint _on_studio

### CODE REVIEW

**Comment by carsongee** [comment]
@bdero can you take a look at this and see if there are any other quality issues you'd like to see cleaned up? I don't feel very qualified on JS style.

File: edx_sga/static/js/src/edx_sga.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ function StaffGradedAssignmentXBlock(runtime, element) {
         function renderStaffGrading(data) {
             $(".grade-modal").hide();
 
-            if (data.display_name != ""){
+            if (data.display_name !== ""){
```
Comment (bdero) [inline edx_sga/static/js/src/edx_sga.js]: Nit: Please put a space after the argumet list:
```javascript
if (data.display_name !== "") {
```

**Comment by bdero** [comment]
Looks great after my one minor comment is addressed! :bookmark:

**Comment by bdero** [comment]
Actually, I think there are actually other improvements that can be made to this code - it's just not easy to comment on them from the diff view in this PR. I'll list them here.

* For one, the vars at the top can be comma separated instead of being repeated var var statements: https://github.com/arbisoft/edx-sga/blob/fix_js_lint_issues/edx_sga/static/js/src/edx_sga.js#L4-L14

* This line looks awkward: https://github.com/arbisoft/edx-sga/blob/fix_js_lint_issues/edx_sga/static/js/src/edx_sga.js#L20
It looks like it's really trying to do this:
```javascript
state.error = state.error || false;
```

* https://github.com/arbisoft/edx-sga/blob/fix_js_lint_issues/edx_sga/static/js/src/edx_sga.js#L61
On this line, an `===` should be used instead of `==` to lessen the likelihood of subtle bugs. It's strange that JSHint isn't catching this.



If you haven't seen this document yet, it's a good quick read:
https://github.com/edx/edx-platform/wiki/Javascript-standards-for-the-edx-platform
The practices listed there are common. As noted, edX follows the Google JavaScript Style guide (which IMO is a very nice and predictable style to go by).

**I recommend installing a linter for Google's Style Guide and using that alongside JSHint.** I just found one by poking around and tried it out; it's properly catching lots of Style issues pertaining to whitespace, double vs single quotes, and line length:
```bash
# Install it globally using pip for python2
sudo pip2 install closure-linter
# Run it, passing the directory in which to search for JavaScript
gjslint -r edx-sga/
```

Let me know if you run into any trouble or have any questions.

**Comment by amir-qayyum-arbisoft** [comment]
@bdero done with improvements, I ran gjslint on code and fixed all issues, it identified. However there are 8 line greater than 80 chars errors, Which seems wrong, because of indentation (tabs) some lines are more then 80 chars, i think it is OK, Any comment?

**Comment by bdero** [comment]
Thanks Amir. Those errors about line length are actually correct, we prefer not to exceed 80 characters for line length (including indentation spaces) for readability purposes.

File: edx_sga/static/js/src/edx_sga.js [hunk 2]
```diff
@@ -48,42 +53,42 @@ function StaffGradedAssignmentXBlock(runtime, element) {
...
-                    if (data.jqXHR.status == 413) {
+                    if (data.jqXHR.status === 413) {
                         /* I guess we have no way of knowing what the limit is
                          * here, so no good way to inform the user of what the
                          * limit is.
                          */
-                        state.error = "The file you are trying to upload is too large."
+                        state.error = 'The file you are trying to upload is too large.';
                     }
                     else {
```
Comment (bdero) [inline edx_sga/static/js/src/edx_sga.js]: Even though `gjslint` isn't catching it, another helpful thing in this PR would be to format curly braces in the style mentioned here: https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml#Code_formatting

Notice that curly braces in this file are formatted like:
```javascript
if (condition) {
    // execute if condition
}
else {
    // execute if not condition
}
```

Where the style guide recommends:
```javascript
```javascript
if (condition) {
    // execute if condition
} else {
    // execute if not condition
}
```
```

File: edx_sga/static/js/src/edx_sga.js [hunk 3]
```diff
@@ -1,44 +1,49 @@
...
+        ),
+        staffDownloadUrl = runtime.handlerUrl(element, 'staff_download'),
+        staffAnnotatedUrl = runtime.handlerUrl(
+          element, 'staff_download_annotated'
+        ),
+        staffUploadUrl = runtime.handlerUrl(element, 'staff_upload_annotated'),
+        enterGradeUrl = runtime.handlerUrl(element, 'enter_grade'),
+        removeGradeUrl = runtime.handlerUrl(element, 'remove_grade'),
+        template = _.template($(element).find('#sga-tmpl').text()),
+        gradingTemplate;
```
Comment (bdero) [inline edx_sga/static/js/src/edx_sga.js]: On second thought, these actually seem a lot more readable as separate `var` statements... Sorry. What do you think @carsongee ?

Comment (amir-qayyum-arbisoft) [inline edx_sga/static/js/src/edx_sga.js]: yeah you are right, I will fix that

Comment (carsongee) [inline edx_sga/static/js/src/edx_sga.js]: Yeah, I've never been a big fan of comma separated var declarations with initialization.  If you aren't initializing anything they work fine.

**Comment by amir-qayyum-arbisoft** [comment]
@bdero  done with fix

**Comment by bdero** [comment]
Thanks Amir - this looks good to me :inbox_tray:

Second review @carsongee ?

**Comment by carsongee** [comment]
:shipit:

**Pull request merged**